src/lib/Scenes/Artwork/Components/AboutArtist.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Sans } from "@artsy/palette"
+import { AboutArtist_artwork } from "__generated__/AboutArtist_artwork.graphql"
+import { ArtistListItemContainer as ArtistListItem } from "lib/Components/ArtistListItem"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface AboutArtistProps {
+  artwork: AboutArtist_artwork
+}
+
+export class AboutArtist extends React.Component<AboutArtistProps> {
+  render() {
+    return (
+      <Flex alignItems="flex-start" mt={2}>
+        <Sans size="4t" weight="medium" mb={2}>
+          About the artist
+        </Sans>
+        <ArtistListItem artist={this.props.artwork.artist} />
```

Comment:
We talked about this in slack but putting a note that this should iterate through all of the artwork's artists and render a card for each!

Reply:
👍 this should now support multiple artists.